x/feesplit/keeper/msg_server_test.go

```diff
@@ -222,3 +222,5 @@ func (suite *KeeperTestSuite) TestRegisterFee() {
 		})
 	}
 }
+
+// TODO Update and Cancel tests
```

Comment:
add tests or create a ticket as follow up

Reply:
Will be handled in this ticket: https://linear.app/evmos/issue/ENG-451/testing-and-qa